requirements.txt

```diff
@@ -9,3 +9,4 @@ pyyaml==3.11
 python-social-auth==0.2.13
 django-rest-swagger==0.3.4
 django-cors-headers==1.1.0
+git+git://github.com/dlitz/pycrypto@af058ee6f5da391a05275470ab4a4a96aa22b350
```

Comment:
hmmm......looking at it closer.......it seems the project might be dead. Which is sad because I don't like the other crypto libraries.
Still looking into this.....